zebedee-cms/src/main/java/com/github/onsdigital/zebedee/util/EncryptionUtils.java

```diff
@@ -64,4 +68,16 @@ public static InputStream encryptionInputStream(InputStream inputStream, SecretK
     public static InputStream encryptionInputStream(Path path, SecretKey key) throws IOException {
         return encryptionInputStream(Files.newInputStream(path), key);
     }
+
+    public static String createMD5HashSum(String value) {
+        try {
+            byte[] bytesOfMessage = value.getBytes("UTF-8");
+            MessageDigest md = MessageDigest.getInstance("MD5");
+            byte[] theMD5digest = md.digest(bytesOfMessage);
+            return  DatatypeConverter.printHexBinary(theMD5digest).toUpperCase();
+        } catch (UnsupportedEncodingException | NoSuchAlgorithmException e) {
+            throw new RuntimeException(String.format("Failure creating MD5 Hashsum: %s", e.getMessage()));
```

Comment:
Rather than passing the exception message like this, you can also pass the whole exception as a second parameter of `RuntimeException`:
```java
throw new RuntimeException("Error generating md5", e);
```
Having said that, it may be worth to also include the `value` parameter as part of this message, so that it's easier to identify the value that caused the error.

Reply:
have addressed this now